sdk/lib/rtl/heappage.c

```diff
@@ -1297,59 +1332,103 @@ RtlpDphVerifyIntegrity(PDPH_HEAP_ROOT DphRoot)
 }
 
 VOID NTAPI
-RtlpDphReportCorruptedBlock(PDPH_HEAP_ROOT DphRoot,
-                            ULONG Reserved,
-                            PVOID Block,
-                            ULONG ValidationInfo)
+RtlApplicationVerifierStop(
```

Comment:
Move to the NDK? (together with the defines from above)

Reply:
Where in the NDK?

Reply:
Or maybe in sdk/include/reactos/verifier.h ? (until we find a good place in the NDK for adding them: I was originally thinking inside ndk/rtlfuncs.h, but it could as well be of interest to split it up, so I didn't have a definitive answer...)